src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -89,4 +89,17 @@
         btn.dataset.isFormValid = 0;
         btn.addEventListener('click', clickHandler, false);
     });
+
+    global.addEventListener('scroll', () => {
+        doc.getElementById('ez-edit-header-details').style.maxHeight = window.scrollY > 70 ? '0px' : '70px';
+    });
+
+    if (doc.getElementById('ez-detalis-admin-bredcrumb-toggler')) {
+        doc.getElementById('ez-detalis-admin-bredcrumb-toggler').addEventListener('click', (event) => {
```

Comment:
Please use always classes for JS hooks not ids. 
Always use `querySelector`
If you will take the same element twice please assign it to a variable, you will avoid going thru DOM multiple times